.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-- v7.9.0
+- v8.5.0
```

Comment:
I suggest using `lts/*` from https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions.

Reply:
Ah nice, I'd not see that before.